### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
Simple HTML list of To Do tasks.
This simple web page will be built using webpack and served by a webpack dev server.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
-
 import './style.css';
 
-
 const allTasks = document.querySelector('.task-list');
 
 const listArray = [
```
Comment (arslanbisharat) [inline src/index.js:5]: - [x] All the changes to the project are not reflected in the pull request. Let's make sure you have all the changes here.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](...) licensed.
```
Comment (arslanbisharat) [inline README.md:40]: - [x] Please add proper links to your Readme.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# to-do-list
+
```
Comment (arslanbisharat) [inline README.md:2]: - [x] The project has unnecessary files such as .vscode. Let's add that to .gitignore.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Ugboaja-Uchechi** [comment]
Fix all code reviews

**Review by wandji20 — APPROVED** [review]
## Hi @Ugboaja-Uchechi  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this project :+1:

# Status: Approved 👍 :partying_face: 
Your project has been approved, however, I have left some comments to help improve your project.

## Congratulations :+1:  :fireworks: 
## Optional

- [ optional ] It will be nice to have webpack devserver setup and running.

- [ optional ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, .eslintrc.json, etc).

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**